pkgs/development/python-modules/stravaweblib/default.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  arrow,
+  pythonOlder,
+  buildPythonPackage,
+  fetchFromGitHub,
+  stravalib,
+  beautifulsoup4,
+  setuptools,
+  setuptools-scm,
+}:
```

Choose a less random order, and note how I removed `arrow`.

```suggestion
{
  lib,
  buildPythonPackage,
  fetchFromGitHub,
  pythonOlder,
  
  # build-system
  setuptools,
  setuptools-scm,
  
  # dependencies
  stravalib,
  beautifulsoup4,
}:
```

---

Addressed in 57b4a24f4d4d